criu/util.c

```diff
@@ -515,6 +515,27 @@ int cr_system(int in, int out, int err, char *cmd, char *const argv[], unsigned
 	return cr_system_userns(in, out, err, cmd, argv, flags, -1);
 }
 
+static int cmp_keep_on_exec(const void *__a, const void *__b)
+{
```

Comment:
@cyrillos this function is also defined in `criu/config.c`. Could we reuse it to avoid code duplication?

Reply:
I happen to miss it. Will take a look, thanks!

Reply:
[PATCH v2] criu: Add --keep-on-exec option